ogr/services/pagure/service.py

```diff
@@ -244,5 +245,15 @@ def project_create(
         if namespace:
             parameters["namespace"] = namespace
 
-        self.call_api(request_url, "POST", data=parameters)
+        try:
+            self.call_api(request_url, "POST", data=parameters)
+        except PagureAPIException as ex:
+            if (
+                ex.pagure_response
+                and ex.pagure_response["errors"]["namespace"][0] == "Not a valid choice"
+            ):
+                raise OgrException(
+                    f"Cannot create project in given namespace ({namespace})."
```

Comment:
Can we determine these two possibilities?

- non-existing namespace
- no permissions

Reply:
I remember it returning the same thing. _not a valid choice_

Reply:
yeah, returns the same thing

Reply:
Hmm. I'll create an issue for that, so we don't forget about that.

Reply:
#245